gandalf.js

```diff
@@ -2,44 +2,44 @@
  * Extras Plugin
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.